accord-core/src/main/java/accord/messages/TxnRequest.java

```diff
@@ -234,7 +234,7 @@ public static <I, O> O computeScope(Node.Id node, Topologies topologies, I keys,
         O scope = null;
         for (int i = startIndex, mi = topologies.size() ; i < mi ; ++i)
         {
-            Topology topology = topologies.get(i);
+            Topology topology = topologies.get(i).global();
```

Comment:
this was a bug found with remove prefix.  The subset topologies would leak the `global` values and not the subset values, which caused the scope logic to work as it works off `global`, so once this was fixed I had to be explicit and ask for the `global` values...